src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,11 @@
+package core.mate.academy.model;
```

Comment:
Let's create impl package in service package and replace all Machine producers there, they aren't related to model package

Reply:
thanks for the finding mistakes !